### DISCUSSION TITLE
Mixed Debugging

### DISCUSSION THREAD

**Discussion by coder0xff** [fzyzcjy/flutter_rust_bridge#1231] [Show and tell]
If you've ever worked with native interop, be it with Dart or otherwise, you probably recognize the value in debugging code in the virtual machine and debugging native code simultaneously. The idea is that you can be stepping through your Dart code, at which point it calls a native function, and suddenly you're stepping through native code and then back. It's a compelling workflow. While getting the debugger running in just Dart or just Rust is pretty straightforward, getting mixed debugging isn't straightforward yet. I wanted to achieve mixed debugging in VS Code with Dart and Rust in a way that was completely automated and cross-platform.

https://coder0xff.wixsite.com/community/post/mixed-debugging-with-dart-and-rust

**Comment by fzyzcjy** [comment]
It looks helpful to allow mixed debugging! If you like, feel free to PR to add your article into https://cjycode.com/flutter_rust_bridge/article.html as well, since this looks useful to users of frb when they want mixed debugging.
